src/Components/ServiceExtensions.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Microsoft.Framework.DependencyInjection;
+using Microsoft.Framework.DependencyInjection.Extensions;
+using System.Collections.Generic;
+
+namespace Components
+{
+	public static class ServiceExtensions
+	{
+		public static IServiceCollection UseComponentSystem(this IServiceCollection services)
+		{
+			services.TryAdd(DefaultServices());
```

Comment:
Can do this inline. I'm not sure if you would want to make the services public. After all, that's why they are internal.